oslc_prolog/applications/oslc_cliopatria_ui.pl

```diff
@@ -0,0 +1,225 @@
+:- module(oslc_cliopatria_ui, []).
+
+:- http_handler(root(oslc_resource_shapes), oslc_resource_shapes, []).
+:- http_handler(root(oslc_shape_resources), oslc_shape_resources, []).
+:- http_handler(root(oslc_shape), oslc_shape, []).
+
+:- include(bundle(html_page)).
+:- use_module(components(label)).
+
+cliopatria:menu_item(300=places/oslc_resource_shapes, 'Shapes').
```

Comment:
No, this is a weight specifying where to place the 'Shapes' menu in the menu bar. :)